.pylintrc.toml

```diff
@@ -422,6 +422,7 @@ disable = [
   "useless-import-alias",  # MyPy requires the opposite
   "wrong-import-order",  # isort-handled: https://github.com/pylint-dev/pylint/issues/9977
   "wrong-import-position",  # isort-handled: https://github.com/pylint-dev/pylint/issues/9977
+  "relative-beyond-top-level",  # Developer preference
```

Comment:
Looks like this is not a preference, but a hack for suppressing the bug in pylint.